recipe/meta.yaml

```diff
@@ -1,49 +1,35 @@
 {% set version = "1.2.0" %}
 
-{% set variant = "openblas" %}
-
-
 package:
   name: cvxopt
   version: {{ version }}
 
 source:
-  fn: cvxopt-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/c/cvxopt/cvxopt-{{ version }}.tar.gz
   sha256: 3296c9d49b7dcb894b20db5d7d1c1a443912b4d82358e03f836575e8398e0d60
-  patches: cvxopt_win_patch.diff   # [win]
```

Comment:
This patch is currently necessary on Windows to build CVXOPT with FFTW and DSDP.